packages/db/metrics.js

```diff
@@ -30,6 +30,30 @@ export async function getUploadMetrics (client) {
   }
 }
 
+const uploadTypeMapping = {
+  uploads_car_total: 'Car',
+  uploads_blob_total: 'Blob',
+  uploads_multipart_total: 'Multipart',
+  uploads_upload_total: 'Upload'
```

Comment:
The DB client shouldn't be concerned with the keys we're using in the metrics. An actual upload "type" should be passed to `getUploadTypeMetrics`.